src/ResourceManager/Resources/Commands.Resources/ActiveDirectory/NewAzureADSpCredentialCommand.cs

```diff
@@ -132,7 +137,24 @@ public override void ExecuteCmdlet()
                 }
                 else
                 {
-                    throw new InvalidOperationException("No valid keyCredential or passwordCredential to update!!");
+                    // If no credentials provided, set the password to a randomly generated GUID
+                    Password = Guid.NewGuid().ToString().ConvertToSecureString();
```

Comment:
How easy would it be to add tests for the new parameter set for new-spcredential, and to update the tests for both cmdlets to ensure the wrapper is filled out?

Reply:
If we can't add automation this milestone, please file an issue for adding automation next milestone